microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -595,15 +601,25 @@ See also:
 * [Introduction to the Performance tool](../evaluate-performance/index.md)
 
 Status:
-* This checkbox is present in Microsoft Edge Canary 129.
-* This checkbox is present in Microsoft Edge Stable 128.
+* This checkbox is present in Microsoft Edge Canary 131.
+* This checkbox is present in Microsoft Edge Stable 129.
 
 
 <!-- ------------------------------ -->
-#### Enable new Performance panel landing page which includes live metrics and field data
+#### Performance panel: enable server timings in the timeline
 <!-- checkbox has no "(?)" link pointing to this anchor wording -->
 
-This is the older label for the checkbox.  See the above section.
+_new as of Microsoft Edge Stable 129_
+
+In the **Performance** tool, gathers server timing data and displays that data.
```

Comment:
need desc for "Performance panel: enable server timings in the timeline"
@captainbrosset 

https://review.learn.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/experimental-features/?branch=pr-en-us-3283#performance-panel-enable-server-timings-in-the-timeline

Reply:
```suggestion
Shows a **Server Timings** track in the **Main** section of the **Performance** tool, to display server timing events sent by your HTTP server. To learn more, see [Server timing](https://developer.mozilla.org/docs/Web/API/Performance_API/Server_timing) at MDN Web Docs.
```